src/Merchello.Web.UI.Client/src/views/common/dialogs/edit.address.html

```diff
@@ -9,29 +9,29 @@
                 <p class="text-warning">{{ dialogData.warning }}</p>
             </div>
             <div class="form-group row-fluid">
-                <label for="name"><localize key="merchelloCustomer_fullName" />:</label>
+                <label for="name"><localize key="merchelloCustomers_fullName" />:</label>
```

Comment:
@YodasMyDad The localize key needed to be `merchelloCustomers_WHATEVER` instead of `merchelloCustomer_WHATERVER`